playground/mocks/data/idp/idx/enroll-profile-with-password.json

```diff
@@ -0,0 +1,143 @@
+{
+  "stateHandle": "NON_NULL_VALUE",
+  "version": "NON_NULL_VALUE",
```

Comment:
same comment as above, client library will validate version, use a real version here, "1.0.0"

Reply:
Thanks, changed in both places indicated